controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/kvtable/AbstractTableRequestProcessor.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.eventProcessor.requesthandlers.kvtable;
+
+import com.google.common.base.Preconditions;
+import io.pravega.common.concurrent.Futures;
+import io.pravega.controller.eventProcessor.impl.SerializedRequestHandler;
+import io.pravega.controller.server.eventProcessor.requesthandlers.RequestUnsupportedException;
+import io.pravega.controller.store.kvtable.KVTableMetadataStore;
+
+import io.pravega.shared.controller.event.ControllerEvent;
+import io.pravega.shared.controller.event.kvtable.CreateTableEvent;
+import io.pravega.shared.controller.event.kvtable.TableRequestProcessor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ScheduledExecutorService;
+
+/**
+ * Abstract common class for all request processing done over SerializedRequestHandler.
+ * This implements TableRequestProcessor interface and implements failing request processing for all ControllerEvent types with
+ * RequestUnsupported.
+ * Its derived classes should implement specific processing that they wish to handle.
+ *
+ * This class provides a common completion method which implements mechanisms that allow multiple event processors
```

Comment:
this comment is not required here. this doesnt provide the common completion method for fairness.